src/bin/pgcopydb/copydb.h

```diff
@@ -216,8 +216,8 @@ typedef struct CopyDataSpec
 	SourceFilterItem *hOid;     /* hash table of objects, by Oid */
 	SourceFilterItem *hName;    /* hash table of objects, by pg_restore name */
 
-	char source_pguri[MAXCONNINFO];
-	char target_pguri[MAXCONNINFO];
+	char *source_pguri; 	    /* malloc'ed area */
+	char *target_pguri;         /* malloc'ed area */
```

Comment:
Again, just pointers now, the memory allocation does not happen in that structure anymore, right?